### PR TITLE
update CHANGELOG for 17.03.0-ce

### PR DESCRIPTION
**Pull request by vieux**
ping @aaronlehmann @mavenugo @aboch @thaJeztah

### CODE REVIEW

**Comment by aaronlehmann** [comment]
We need to mention and explain the versioning change.

Here are changelog entries for swarm mode changes:

- Fix possibility of tasks getting stuck in the "NEW" state during a leader failover https://github.com/docker/swarmkit/pull/1938
- Fix extraneous task creations for global services that led to confusing replica counts in `docker service ls` https://github.com/docker/swarmkit/pull/1957
- Fix problem that made rolling updates slow when `task-history-limit` was set to 1 https://github.com/docker/swarmkit/pull/1948
- Restart tasks elsewhere, if appropriate, when they are shut down as a result of nodes no longer satisfying constraints https://github.com/docker/swarmkit/pull/1958

And another networking one:

- Fix port allocation when multiple published ports map to the same target port https://github.com/docker/swarmkit/pull/1835

**Comment by vieux** [comment]
@aaronlehmann @mavenugo @thaJeztah PTAL

**Comment by vieux** [comment]
ping @friism

**Comment by friism** [comment]
(not maintainer) lgtm!

**Review by thaJeztah — APPROVED** [review]
LGTM

**Comment by mavenugo** [comment]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,24 +7,41 @@ be found.
 
 ## 17.03.0-ce (2017-03-01)
 
+**IMPORTANT**: Starting with this release, Docker is on a monthly release cycle and uses a
+new YY.MM versioning scheme to reflect this. In addition to a monthly release, more stable
```
Comment (aaronlehmann) [inline CHANGELOG.md]: I wonder if "more stable" is the right terminology here. Even as a maintainer, I don't quite understand this. Are we saying that the monthly releases are bleeding edge, and shouldn't be run by users that value stability, but the quarterly releases will be somehow better? Or do we mean to say that the quarterly releases will be supported for a longer period? (I actually don't know the answer here.)

How can users distinguish a "monthly" release from a "quarterly" release?

Comment (friism) [inline CHANGELOG.md]: Maybe this should say: "Two channels are available: monthly and quarterly. Any given monthly release will only receive security and bugfixes until the next monthly release is available. Quarterly releases receive security and bugfixes for 4 months after initial release."

Comment (vieux) [inline CHANGELOG.md]: I updated PTAL, regarding `How can users distinguish a "monthly" release from a "quarterly" release?` I think we need some sort a graph

**Comment by aaronlehmann** [comment]
LGTM

**Comment by ralphtheninja** [comment]
Hi. Where can I read up on the new versioning scheme?

**Pull request merged**